pages/mesosphere/dcos/services/kafka/2.10.0-2.4.0/security/index.md

```diff
@@ -33,12 +33,144 @@ For more information on {{ model.techShortName }}'s security, read the following
 
 <p class="message--note"><strong>NOTE: </strong>It is possible to update a running DC/OS {{ model.techName }} service to enable transport encryption after initial installation, but the service may be unavailable during the transition. Additionally, your {{ model.techShortName }} clients will need to be reconfigured unless `service.security.transport_encryption.allow_plaintext` is set to true.</p>
 
+### Using Custom TLS certificate
```

Comment:
Custom TLS doc is inline here but the Default TLS option doc seems to be included via  mesosphere/dcos/services/include/security-configure-transport-encryption.tmpl template? Any specific reason for the discrepancy?